### ISSUE TITLE
New comment count - redesign compatibility

### ISSUE THREAD

**Issue by andytuba** [honestbleeps/Reddit-Enhancement-Suite#4954]
Add comment count via CSS

.Post.t3_12345 a[data-click-id="comments"]:after { title: "123 new"; }

Track new comments by pulling data from JSAPI metadata

**Comment by Zshuff03** [comment]
Hey! First time contributing to this repo, but this seems like a pretty simple task, I'll get too it.

**Comment by Zshuff03** [comment]
On second thought, I'm not quite sure how to see if it's redesign or not and where to make this change. If you could point me in a direction that would be awesome 👍

**Comment by Ajedi32** [comment]
I'm not 100% certain, but it seems to me you just need to update the existing code that already does this to account for the DOM changes in the redesign. (Such that it works with both.)

You might want to reference some of the previous changes made concerning the redesign as examples: https://github.com/honestbleeps/Reddit-Enhancement-Suite/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Amerged+%22redesign%22

**Comment by Zshuff03** [comment]
I've noticed that in `newCommentCount`  in the before module load,
`watchForThings(['post'], displayNewCommentCount, { immediate: true });`
is being used for setting the new comment count but this never gets called in the redesign.
I have attempted to use 
`watchForRedditEvents('post', (thing, postEventData) => {...` instead but that only gives me a `thing` span that is empty. Is there another `watchFor` that I am missing?

**Comment by andytuba** [comment]
Yes, you should use `watchForRedditEvents('post', (thing, postEventData) => {` in the redesign.

The comment count should be included in `postEventData`.

**Comment by andytuba** [comment]
_waves magic wand_

Look again!

Could you also add `numComments` to `Post` and `PostModTools` types in `/types/events`?

**Comment by Zshuff03** [comment]
Nice! I'm now running into an issue where the module.afterLoad is not running, therefore never setting the current number of comments, when we hit the comments page, and then `getEntryBatched` is always null. I'll keep searching for the issue but any ideas would be appreciated.
